RegExp.make.js

```diff
@@ -971,11 +971,12 @@ RegExp.make = (function () {
   }
 
   function make(flags, template, ...values) {
+    // make is always bound to a flags string
     if ('string' === typeof template && values.length === 0) {
       // Allow RegExp.make(i)`...` to specify flags.
       // This calling convention is disjoint with use as a template tag
       // since the typeof a template record is 'object'.
-      return make.bind(null, template /* use as flags instead */);
+      return make.bind(null, flags + template /* use as flags instead */);
```

Comment:
I'd prefer to throw immediately. If we know it will fail later, why defer the failure? The use case for currying here includes passing around the "maker" function for later use, so the exception could occur far away from the site of the double-flag.

Of course, another alternative is to uniquify the code units (perhaps `[...new Set(flags + template)].join('')`) in `make`.